src/Chirp.Razor/CheepRepository.cs

```diff
@@ -3,19 +3,22 @@
 using CheepRecord;
 using Initializer;
 
-public class CheepRepository
+
+namespace Repository;
+
+public static class CheepRepository
 {
-    private ChirpDBContext db;
+    private static readonly ChirpDBContext db;
 
-    public CheepRepository()
+    static CheepRepository()
```

Comment:
The reason it was static is that i forgot to change it back. I used static to test the methods without the need to make a class since i was not sure where to put it i will change this back so it's easier to test with. Thanks for catching this :D

Reply:
Nice. Thanks for the reply. Will you make a new commit on the branch? It will automatically add the commit to the current pull request.